style.css

```diff
@@ -54,6 +54,8 @@ div.navbar-collapse > div.navbar-nav > .nav-link {
 
 main {
 	margin-top: 2rem;
+	margin-top: 3rem;
```

Comment:
While resolving the merge conflicts, this should be removed. Gigishka changed the line `margin-top: 3rem;` to `margin-top: 2rem;`, but Kethas, who did not have Gig's version, committed with the `3rem` instead of the `2rem` and it conflicted.